### PR TITLE
Fix compaction style in tikv configuration file

### PR DESCRIPTION
**Pull request by tabokie**
### What is changed, added or deleted? (Required)

Refer to https://github.com/tikv/tikv/issues/11028

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tikv-configuration-file.md [hunk 1]
```diff
@@ -1117,8 +1117,8 @@ Configuration items related to `rocksdb.defaultcf`, `rocksdb.writecf`, and `rock
 ### `compaction-style`
 
 + Compaction method
-+ Optional values: `"level"`, `"universal"`
-+ Default value: `"level"`
++ Optional values: `0` (`Level`), `1` (`Universal`), `2` (`Fifo`)
++ Default value: `0`
```
Comment (TomShawn) [inline tikv-configuration-file.md:1121]: 这里是否可看作是兼容性变化呢？

Comment (tabokie) [reply tikv-configuration-file.md:1121]: This PR only applies to versions <= v5.2.0. And it's a bugfix, not a change on our end.

**Review by Connor1996 — APPROVED** [review]
LGTM

File: tikv-configuration-file.md (hunk 1 above)
Comment (TomShawn) [reply tikv-configuration-file.md:1121]: What is the relationship between 0 and Level, 1 and Universal, 2 and Fifo? Are they equal to each other?

Comment (tabokie) [reply tikv-configuration-file.md:1121]: Only number is accepted, name is for explanation.

File: tikv-configuration-file.md [hunk 2]
```diff
@@ -1117,8 +1117,8 @@ Configuration items related to `rocksdb.defaultcf`, `rocksdb.writecf`, and `rock
 ### `compaction-style`
 
 + Compaction method
-+ Optional values: `"level"`, `"universal"`
-+ Default value: `"level"`
++ Optional values: `0` (`Level`), `1` (`Universal`), `2` (`Fifo`)
```
Comment (en-jin19) [inline tikv-configuration-file.md:1120]: @tabokie @TomShawn how about adding this description?

```suggestion
+ Optional values (only numbers can be accepted): `0` (`Level`), `1` (`Universal`), `2` (`Fifo`)
```

Comment (tabokie) [reply tikv-configuration-file.md:1120]: No objection.

Comment (TomShawn) [reply tikv-configuration-file.md:1120]: LGTM

**Review by en-jin19 — APPROVED** [review]
LGTM

**Pull request merged**